kontrakter/felles/kodeverk/pom.xml

```diff
@@ -0,0 +1,27 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>kodeverk</artifactId>
+    <packaging>jar</packaging>
+    <name>Kodeverk ESAS</name>
+    <description>JSON-schema for kodeverk</description>
+
+    <parent>
+        <groupId>io.github.domstolene.esas</groupId>
+        <artifactId>kodeverk</artifactId>
+        <version>tmp</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
+    <build>
+        <resources>
+            <resource>
+                <targetPath>schema</targetPath>
+                <directory>./</directory>
+                <includes>
+                    <include>**/*.json</include>
+                </includes>
```

Comment:
Caset her blir jo litt annerledes enn json-schema vi har for meldinger. Vi har ikke behov for å validere ved mottak/sending av dette schema, da vi ikke sender eller mottar det. Så er sånn sett unødvendig å dytte opp json-schema 🤔

Reply:
Fjerner pom fil.